proto/data_test.go

```diff
@@ -219,11 +219,11 @@ func TestPrevKey(t *testing.T) {
 }
 
 func TestKeyString(t *testing.T) {
-	if KeyMax.String() != "\xff..." {
+	if KeyMax.String() != "\\xff..." {
```

Comment:
on a related note, `key.String()` should probably replace the `bytes.Index` with a `bytes.Equal` or update its description:
```go
// String returns a string-formatted version, with a maximum
// key formatted for brevity as "\xff...".
func (k Key) String() string {
	if idx := bytes.Index(k, KeyMax); idx != -1 {
		return string(MakeKey(k[:idx], Key("\xff..."), k[idx+KeyMaxLength:]))
	}
	return string(k)
}
```